### ISSUE TITLE
TAMA/AKATSUKI /AOSP11 / problem battery consumption 

### ISSUE THREAD

**Issue by NYGGAZ** [sonyxperiadev/bug_tracker#644]
Platform: tama
Device: h9436
Kernel version: 4.14.196-g7aaaf731280bc-dirty
#1 Thu Oct 8 12:51:35 CEST 2020
Android version: 11
Software binaries version: SW_binaries_for_Xperia_Android_11_4.14_v1a_tama.

**Description**

the battery consumes extremely quickly. more than normal. los17.1 was much more enduring. I have normal use.  no games, a few videos and a few messages only.  nothing more.

**Comment by leeoniya** [comment]
maybe related: #581

**Comment by NYGGAZ** [comment]
> maybe related: #581 

yep.  Thank you😉.  yes I saw that.  but personally I had no problem with battery on android 10. neither stock nor sodp.  with LOS 17.1 sodp I had very good battery life.

**Comment by NYGGAZ** [comment]
> maybe related: #581 

Oh yes.  also, i wanted to apologize as this is the first time that bug reports have been made here.  so not the habit.  I hope I have done well.  over time, I will improve my bug reports

**Comment by DesantBucie** [comment]
I think i had similar feelings. I had AOSP 11 installed two times and it was no better than normal. Now i have AOSP10 and since morning i still have the same procentage(not using phone, but still AOSP11 would consume few percents)

**Comment by jerpelea** [comment]
Discontinued Android version
